pkg/email/send_email.go

```diff
@@ -29,7 +30,14 @@ type Params struct {
 	LakefsBaseURL      string
 }
 
-func NewEmailer(p Params) *Emailer {
+func NewEmailer(p Params) (*Emailer, error) {
+	if p.SMTPHost == "" {
+		return &Emailer{}, nil
```

Comment:
what will happen if we send an email using this Emailer when there is no Dialer? calling a method on Nil pointer.

Reply:
Good point, it will be a nil pointer. FIxed and testing the functionalities once again to verify it works